src/WebUI/Startup.cs

```diff
@@ -25,6 +26,32 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddApplication();
             services.AddInfrastructure(Configuration);
+
+            services.AddHttpContextAccessor();
+
+            services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = "cookie";
+                options.DefaultSignInScheme = "cookie";
+                options.DefaultChallengeScheme = "oidc";
+            })
+            .AddCookie("cookie")
+            .AddOpenIdConnect("oidc", options =>
+            {
+                options.Authority = "http://localhost:5000";
+                options.RequireHttpsMetadata = false; // dev only
+                options.ClientId = "codidact_client";
+                options.ClientSecret = "acf2ec6fb01a4b698ba240c2b10a0243";
```

Comment:
Is this okay to be kept in public Git history, or should it be made a configurable value?

Reply:
I'll move this to the app.settings file but leave it there as an example. 
Just don't use this specific secret in production

Reply:
Fixed in https://github.com/codidact/core/pull/29/commits/91a0e356feb425644914546816d9229b1356ad61